cmd/soroban-rpc/internal/methods/health.go

```diff
@@ -35,6 +42,11 @@ func NewHealthCheck(txStore *transactions.MemoryStore, maxHealthyLedgerLatency t
 				Message: msg,
 			}
 		}
-		return HealthCheckResult{Status: "healthy"}, nil
+		result := HealthCheckResult{
+			Status:       "healthy",
+			LatestLedger: ledgerRange.LastLedger.Sequence,
```

Comment:
what is recommended usage of this range from readiness probes to know the 24 hour window is complete, can it just subtract i.e. `latest - oldest >  {max_ledgers_per_day}` ?

Reply:
When having ingested the full window `latest - oldest == max(events_window, transactions_window)`. By default `latest - oldest == 17280`.

```
// DefaultEventLedgerRetentionWindow represents the max number of ledgers we would like to keep
// an incoming event in memory. The value was calculated to align with (roughly) 24 hours window.
const DefaultEventLedgerRetentionWindow = 17280
```

I could expose the retention window in the endpoint if that helps. It should make it simpler to compute with jq

Reply:
I just added the retention window to the endpoint. Now you just need to check `latest - oldest = ledgerRetentionWindow`

Reply:
nice, thanks!